packages/rolldown/src/utils/normalize-input-options.ts

```diff
@@ -11,7 +11,7 @@ export async function normalizeInputOptions(
   config: InputOptions,
 ): Promise<NormalizedInputOptions> {
   const { input, ...rest } = config
-  let plugins = await normalizePluginOption(config.plugins)
+  let plugins = (await normalizePluginOption(config.plugins)).flat()
```

Comment:
I thought we have this before.

https://github.com/rolldown/rolldown/blob/38a2c3e6a9d782efdea45a71d15cae589647ea37/packages/rolldown/src/utils/normalize-plugin-option.ts